### PR TITLE
Check if Binary Tree is Symmetric in java

### PR DESCRIPTION
**Pull request by Nidhideveloper**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Added Is Symmetric Tree 

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#6464

### CODE REVIEW

**Comment by Nidhideveloper** [comment]
@HarshCasper Sir kindly review the pr and let me know if there is any mistake that i need to correct.

File: Java/README.md [hunk 1]
```diff
@@ -327,7 +327,7 @@ _add list here_
 - [Find first missing positive number](other/FirstMissingPositiveNumber.java)
 - [First non-repeating character in a string](other/FirstNonRepeatingChar.java)
 - [Insert Position](other/InsertPosition.java)
-- [Keys and rooms for entry](other/Key_n_room.java)
```
Comment (HarshCasper) [inline Java/README.md:330]: Why did you change this when its not a refactor?

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Comment by Nidhideveloper** [comment]
@HarshCasper  Sir, I have made changes accordingly , please see to it. Thank you.

**Comment by Nidhideveloper** [comment]
> @HarshCasper Sir kindly review the pr and let me know if there is any mistake that i need to correct.

Sir mad e changes accordingly.

**Review by ankitaggarwal23 — APPROVED** [review]

**Pull request merged**